### PR TITLE
Update gradle image to use Gradle6; Add user to /etc/group

### PR DESCRIPTION
**Pull request by svor**
Signed-off-by: svor <vsvydenk@redhat.com>

### What does this PR do?
Updates **che-java11-gradle** image to use Gradle v.6.2.1;
Patches `/etc/group` by adding arbitrary user

@amisevsk @nickboldt I added current arbitrary user into /etc/group because I got a message after opening the terminal:
` groups: cannot find name for group ID 1000160000`
Any concerns of that?

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/15723

### CODE REVIEW

**Comment by svor** [comment]
@amisevsk you can try this devfile https://gist.githubusercontent.com/svor/0fe771b1f1e853a98ee38442b75023bd/raw/f9c994c4316927cfb1fc57b37eb9beaf069a4dba/GradleDevfile.

**Comment by tsmaeder** [comment]
@svor I think we need to understand how groups are used before doing the groups change. If the error message (group has no name) existed before, I propose we reduce the PR to just the version update.

**Comment by svor** [comment]
@tsmaeder the error message didn't exist before i can reproduce this message only with new gradle image (https://hub.docker.com/r/amd64/gradle)

File: arbitrary-users-patch/base_images [hunk 1]
```diff
@@ -1,7 +1,7 @@
 che-cpp-rhel7           registry.access.redhat.com/devtools/llvm-toolset-rhel7
 che-dotnet-2.2          mcr.microsoft.com/dotnet/core/sdk:2.2-stretch
 che-golang-1.12         golang:1.12-stretch
-che-java11-gradle       gradle:5.2.1-jdk11
+che-java11-gradle       gradle:6.2.1-jdk11
```
Comment (amisevsk) [inline arbitrary-users-patch/base_images]: The corresponding meta.yaml for gradle should be updated, since it still refers to 5.2.1 in description.

**Comment by amisevsk** [comment]
Looking into the groups issue some more, I have no idea what's going on. On the gradle 5.2.1 image, I have:
```bash
user@workspacefjanf44yq6wpgnro:/projects$ id
uid=1000420000(user) gid=0(root) groups=0(root),1000420000
```
The group has no name, but the message is not printed.

On 6.2.1:
```bash
user@workspacehzwccyj8cswa4iqh:/projects$ id
uid=1000420000(user) gid=0(root) groups=0(root),1000420000
```
and the error message. :man_shrugging:

File: arbitrary-users-patch/base_images (hunk 1 above)
Comment (svor) [reply arbitrary-users-patch/base_images]: fixed

**Pull request merged**